src/Harald.Infrastructure.Slack/SlackFacade.cs

```diff
@@ -26,17 +27,23 @@ public class SlackFacade : ISlackFacade
         private readonly HttpClient _client;
         private readonly IDistributedCache _cache;
         private readonly SlackOptions _options;
+        private readonly string _botUserId;
 
         public SlackFacade( HttpClient client = null, IOptions<SlackOptions> options = null, IDistributedCache cache = null)
         {
             _client = client ?? new HttpClient() { BaseAddress = new System.Uri("https://slack.com", System.UriKind.Absolute) };
             _cache = cache;
             _options = options?.Value;
+            _botUserId = Environment.GetEnvironmentVariable("SLACK_API_BOT_USER_ID") ?? "";
```

Comment:
Please use the configuration system for this. We have already enabled the Env loader. :) Either add the key you need (SLACK_API_BOT_USER_ID) as a Property to the SlackOptions or inject the IConfigration interface and use the GetValue<T>(IConfiguration, String) method to fetch your value (which will ultimately resolve to the env var).

Reply:
Check e9c5937 and 1094b00